### PR TITLE
Edits to training guide

### PR DESCRIPTION
**Pull request by rachelsun97**
Removed deprecated functions and added example of custom units dataframe to edit_attributelists.Rmd

### CODE REVIEW

File: workflows/edit_eml/edit_attributelists.Rmd [hunk 1]
```diff
@@ -105,13 +113,22 @@ If you have units that are not in the standard EML unit list, you will need to b
...
+    
+  id = c('siemensPerMeter', 'decibar'),
+  unitType = c('resistivity', 'pressure'),
+  parentSI = c('ohmMeter', 'pascal'),
+  multiplierToSI = c('1','10000'),
+  abbreviation = c('S/m','decibar'),
+  description = c('siemens per meter', 'decibar'),
+  
+  stringsAsFactors = FALSE)
 ```
```
Comment (dmullen17) [inline workflows/edit_eml/edit_attributelists.Rmd]: This is looking really good.  I think you could add a line where you insert this into the EML doc.

Comment (dmullen17) [reply workflows/edit_eml/edit_attributelists.Rmd]: I actually see that in section 4.3.4 but I think it makes sense to just move it up to the bottom of this section so it's more self-contained

**Comment by dmullen17** [comment]
@rachelsun97 I really like the documentation you wrote for `recover_failed_submission`.  Could you move this from editing eml to [here](https://nceas.github.io/datateam-training/reference/adc-web-submissions.html) instead? I think that's a more appropriate spot for it.  Otherwise looks great!

**Comment by rachelsun97** [comment]
@dmullen17 I moved ```recover_failed_submission``` to the ```ADC web submissions``` section, and adjusted the custom units section according to your earlier comments

**Comment by dmullen17** [comment]
@rachelsun97 this all looks good to me!  I'll merge it in now.

**Pull request merged**